styles.css

```diff
@@ -51,6 +52,47 @@ header nav a {
   color: inherit;
 }
 
+.close {
+  display: none;
+  cursor: pointer;
+}
+
+.close:hover {
+  cursor: pointer;
+}
+
+.close.active {
+  display: block;
+}
+
+.hamburger.active {
+  display: none;
+}
+
+.nav-links {
+  display: block;
+  position: absolute;
+  right: 100%;
+  top: 60px;
+  margin: 16px 0;
+}
+
+.nav-links > li a {
+  font-size: 0.92rem;
+}
+
+.nav-links.active {
+  right: 30%;
+}
+
+.wrapper {
+  margin-top: 10px;
+}
+
+.wrapper.active {
+  margin-top: 120px;
+}
```

Comment:
- [ ] Your menu works fine, but you have to work on your design to match the Figma design. You have to make the background white and just display the close button and the 3 links.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/157056394-35db04c5-926b-40d3-8ff1-91d8696b004d.png) | ![image](https://user-images.githubusercontent.com/35535130/157056525-7dd804a5-57dc-4f8d-bfa1-0a70854c282c.png) |